modules/__tests__/getParams-test.js

```diff
@@ -28,23 +28,7 @@ describe('getParams', function () {
     })
 
     describe('and the pattern is optional', function () {
-      const pattern = '/comments/(:id)/edit'
-
-      describe('and the path matches with supplied param', function () {
-        it('returns an object with the params', function () {
-          expect(getParams(pattern, '/comments/123/edit')).toEqual({ id: '123' })
-        })
-      })
-
-      describe('and the path matches without supplied param', function () {
-        it('returns an object with an undefined param', function () {
-          expect(getParams(pattern, '/comments//edit')).toEqual({ id: undefined })
-        })
-      })
-    })
-
-    describe('and the pattern and forward slash are optional', function () {
-      const pattern = '/comments(/:id)/edit'
+      const pattern = '/comments/:id?/edit'
```

Comment:
I think the change from parens to the question mark for optional params is going to be the biggest issue with this. I believe it was possible to do something like `/user(s)` before, so you might run into that too.